lib/amorail/client.rb

```diff
@@ -4,49 +4,50 @@
 require 'active_support'
 
 module Amorail
+  # Amorail http client
   class Client
-
     attr_accessor :cookies
 
     def initialize
       @host = Amorail.config.api_endpoint
-      @connect = Faraday.new(url: @host) do |faraday|           
-        faraday.adapter  Faraday.default_adapter
-        faraday.response :json, :content_type => /\bjson$/
+      @connect = Faraday.new(url: @host) do |faraday|
+        faraday.adapter Faraday.default_adapter
+        faraday.response :json, content_type: /\bjson$/
         faraday.use :instrumentation
       end
-
     end
 
     def connect
       @connect || self.class.new
     end
-    
+
     def authorize
-      response = post(Amorail.config.auth_url, {'USER_LOGIN' => Amorail.config.usermail, 'USER_HASH' => Amorail.config.api_key})
+      response = post(
+        Amorail.config.auth_url,
+        'USER_LOGIN' => Amorail.config.usermail,
+        'USER_HASH' => Amorail.config.api_key
+      )
       cookie_handler(response)
       response
     end
-    
-    def safe_request(method, url, params={}) 
-      self.send(method, url, params)
-    rescue ::Amorail::AmoUnauthorizedError => e
+
+    def safe_request(method, url, params = {})
+      send(method, url, params)
+    rescue ::Amorail::AmoUnauthorizedError
       authorize
-      self.send(method, url, params)
+      send(method, url, params)
     end
 
-    def get(url, params={})
-      response = connect.get(url) do |request|
-        request.headers['Cookie'] = self.cookies if self.cookies.present?
-        request.headers['Content-Type'] = 'application/json'
-        request.body = params.to_json
+    def get(url, params = {})
+      response = connect.get(url, params) do |request|
+        request.headers['Cookie'] = cookies if cookies.present?
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.